dhis-2/dhis-support/dhis-support-artemis/src/main/java/org/hisp/dhis/artemis/audit/listener/PostDeleteAuditListener.java

```diff
@@ -89,6 +92,21 @@ public boolean requiresPostCommitHanding( EntityPersister entityPersister )
     @Override
     public void onPostDeleteCommitFailed( PostDeleteEvent event )
     {
-        log.debug( "onPostDeleteCommitFailed: " + event );
+        log.info( "onPostDeleteCommitFailed:event: " + event );
+        log.info( "onPostDeleteCommitFailed:id: " + event.getId() );
+
+        try
+        {
+            if ( event.getEntity() != null )
+            {
+                log.info( "onPostDeleteCommitFailed:entity.class: " + event.getEntity().getClass() );
+                log.info( "onPostDeleteCommitFailed:entity: "
```

Comment:
You could log this as one line using https://www.slf4j.org/faq.html#logging_performance using parameterized messages. parameterized messages are generally preferred over string concatenation, see link.

Reply:
Hi @teleivo this was just a special branch for testing out on our performance servers, it will be closed soon